python/GafferSceneUI/SceneViewUI.py

```diff
@@ -86,6 +86,10 @@ def __rendererPlugActivator( plug ) :
 	"toolbarLayout:customWidget:CenterRightSpacer:section", "Top",
 	"toolbarLayout:customWidget:CenterRightSpacer:index", -2,
 
+	"toolbarLayout:customWidget:RightEditScopeBalancingSpacer:widgetType", "GafferSceneUI.SceneViewUI._RightEditScopeBalancingSpacer",
```

Comment:
Could we add a todo about how we think this is horrendous, and we should give PlugLayout some features for arranging plugs in horizontal sections with alignment?

Reply:
Horrendous indeed! Todo added in 6cfe186312659b0a5ac3698c2a7c659fae7d065c